src/test/java/org/testeditor/fixture/host/HostDriverFixtureTest.java

```diff
@@ -1,17 +1,64 @@
 package org.testeditor.fixture.host;
 
+import org.junit.Assert;
 import org.junit.Test;
+import org.testeditor.fixture.host.s3270.Status;
+import org.testeditor.fixture.host.s3270.options.TerminalMode;
+import org.testeditor.fixture.host.s3270.statusformat.ConnectionState;
+import org.testeditor.fixture.host.s3270.statusformat.EmulatorMode;
+import org.testeditor.fixture.host.s3270.statusformat.FieldProtection;
+import org.testeditor.fixture.host.s3270.statusformat.KeyboardState;
+import org.testeditor.fixture.host.s3270.statusformat.ScreenFormatting;
 
 public class HostDriverFixtureTest {
 
+  private final int STANDARD_ROW = Integer.parseInt(System.getenv("STANDARD_ROW"));
+  private final int STANDARD_COLUMN = Integer.parseInt(System.getenv("STANDARD_COLUMN"));
+  private final int MAX_ROWS = 24;
+  private final int MAX_COLUMNS = 80;
+  private final String STANDARD_WINDOW_ID = "0x0";
+
   @Test
   public void connectionTest() {
     String S2370_PATH = System.getenv("S3270_PATH");
     String hostUrl = System.getenv("HOST_URL");
     int hostPort = Integer.parseInt(System.getenv("HOST_PORT"));
+    HostDriverFixture hdf = new HostDriverFixture();
+    hdf.connect(S2370_PATH, hostUrl, hostPort);
+    hdf.disconnect();
+
+  }
+
+  @Test
+  public void statusTest() {
+
+    // given
+    String S2370_PATH = System.getenv("S3270_PATH");
+    String hostUrl = System.getenv("HOST_URL");
+    int hostPort = Integer.parseInt(System.getenv("HOST_PORT"));
 
     HostDriverFixture hdf = new HostDriverFixture();
     hdf.connect(S2370_PATH, hostUrl, hostPort);
+
+    // when
+    Status status = hdf.getStatus();
+
+    // then
+    Assert.assertTrue(status.getKeyboardState().name().equals(KeyboardState.UNLOCKED.name()));
```

Comment:
is there a special reason why comparison is done after "name()" is executed on both values?